### ISSUE TITLE
predict txt is different between eval_rec.py and infer_rec.py

### ISSUE THREAD

**Issue by maxh2010** [Topdu/OpenOCR#65]
I found that  replace_batchnorm is call after model file is loaded in infer_rec.py(line:222), that is caused the pred_result is different
eval_rec.py between infer_rec.py.

why the replace_batchnorm function is added in infer_rec.py

**Comment by Topdu** [comment]
This is a result of the reparameterization technique, and batchnorm allows for convolutional operator merging. The detailed structure of the network can be found in RepViT.
You can give a sample with inconsistent results. Theoretically, replace_batchnorm will only affect inference speed, not accuracy.

**Comment by maxh2010** [comment]
I have a project  to recognize Captcha, the images are cropped for Captcha images , these crop imges are detected and cropped by  detect model.
The detect model is trained by Yolo usring our own data.
![0anRvlHA](https://github.com/user-attachments/assets/773a66df-cb67-458e-81fa-fa67a960f609)
![0AntI9SV](https://github.com/user-attachments/assets/03d466cd-1278-48fb-9c6b-4631a895bb42)
![0AQV37hS](https://github.com/user-attachments/assets/72ef2df0-7754-45f2-b706-514a52746cb2)

**Comment by Topdu** [comment]
After our experiment, the third example you provided appears to have inconsistent results. In cases where the difference in data distribution between the test set and the training set is not significant, replace_norm is not affecting the recognition results. The few examples you have shown have a large difference in dataset distribution from the training set, which the recognition model is unable to recognize correctly, and thus may result in replace_norm affecting the recognition results.
